sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -25,7 +25,8 @@ export enum CredentialType {
 
 export enum ProgramLengthOptions {
   WeeksToLessThanYear = "12WeeksToLessThan1Year",
-  OneToTwoYears = "1YearToLessThan2Years",
+  WeeksTo59Weeks = "52WeeksTo59Weeks",
+  WeeksToTwoYears = "60WeeksToLessThan2Years",
```

Comment:
Since the change only applies to FT applications, we should probably keep the original values for PT applications. Can you verify with the business team?

Reply:
We don't need to keep the original values since the change (for the dropdown) is required prior to Part-Time release.